addons/website/views/new_page_template_views.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<!-- This template is used to obtain the order and the translations of all
+group names -->
+<record id="new_page_template_groups" model="ir.ui.view">
+    <field name="name">New page template groups</field>
+    <field name="type">qweb</field>
+    <field name="key">website.new_page_template_groups</field>
+    <field name="arch" type="xml">
+        <div id="about">About</div>
+        <div id="landing">Landing Pages</div>
+        <div id="gallery">Gallery</div>
+        <div id="services">Services</div>
+        <div id="pricing">Pricing Plans</div>
+        <div id="team">Team</div>
+    </field>
```

Comment:
Can't this just be hardcoded in python?
Is the goal doing it this way so that themes could add their own new section?
I am not sure to understand the need

Reply:
Indeed, it is to allow themes to add sections through xpath.
It was done with a template instead of using some manifest entry in order to support translations.

Reply:
But it could have just been a python method that would return a list of translated terms, right?
It's just not possible because if we import a theme through Odoo module upload mechanism, it doesn't consider the python parts, right?
Hm..